models/oneinch/ethereum/oneinch_v1_ethereum_trades.sql

```diff
@@ -30,218 +31,218 @@
     ,'contract_address'
 ] %}
 
-WITH oneinch_calls AS
-(
+WITH
+  oneinch_calls AS (
     SELECT
-        call_block_number as block_number,
-        CAST(NULL as string) as taker,
-        fromToken as from_token,
-        toToken as to_token,
-        tokensAmount as from_amount,
-        minTokensAmount as to_amount,
-        call_tx_hash as tx_hash,
-        call_block_time as block_time,
-        call_trace_address as trace_address,
-        CAST(-1 as integer) AS evt_index,
-        contract_address
+      call_block_number AS block_number,
+      TRY_CAST(NULL AS VARCHAR) AS taker,
```

Comment:
was `try_cast` preferred in harmonizer?

Reply:
also, `taker` and `maker` fields should always be `varbinary`

Reply:
Yeah the harmonizer sometimes uses `try_cast` rather than cast

Reply:
I will take some time maybe to look into the harmonizer code and change it up so it always tries t make `taker` and `maker` varbinary rather than `varchar`